src/code128.ps.src

```diff
@@ -60,6 +60,8 @@ begin
     /newencoder false def
     /parse false def
     /parsefnc false def
+    /suppressc false def          % Suppress code set C
+    /unlatchextbeforec false def  % Suppress extended ASCII with code set C
```

Comment:
Would you like to suggest text for the wiki docs for what the need arises for `unlatchextbeforec`.

Reply:
I saw it suggested somewhere. There might not be an actual need. We can delete it. It's easy to put back in.

Reply:
I have alternative suggestions for the name "suppressc": "no-c" or "no-set-c" with or without hyphens. I only suggest them because they're shorter.

Reply:
Leave it in. I'll leave it undocumented in the wiki and we'll see if anyone ever encounters the issue.

I've only see FNC4 used with Code 128 in certain Russian automotive use cases.